release/windows-installer/src/WindowsInstaller.hs

```diff
@@ -13,18 +13,28 @@ import SdkVersion
 
 main :: IO ()
 main = do
-    [installerFile, sdkDir] <- getArgs
-    writeFile installerFile $ nsis $ installer sdkDir
+    [installerFile, sdkDir, logo] <- getArgs
+    writeFile installerFile $ nsis $ installer sdkDir logo
 
-installer :: FilePath -> Action SectionId
-installer sdkDir = do
-    name "DAML SDK installer"
+installer :: FilePath -> FilePath -> Action SectionId
+installer sdkDir logo = do
+    name "DAML SDK"
     outFile "daml-sdk-installer.exe"
+    unsafeInjectGlobal $ "!define MUI_ICON \"" <> logo <> "\""
     requestExecutionLevel User
+    unsafeInjectGlobal $ "!insertmacro MUI_PAGE_WELCOME"
+    page InstFiles
+    page $ Finish finishOptions
+        { finLinkText = "Open the DAML Quickstart guide"
+        , finLink = "https://docs.daml.com/getting-started/quickstart.html"
+        }
     section "" [] $ do
         -- We use PLUGINSDIR as an easy way to get a temporary directory
         -- that nsis will cleanup automatically.
         unsafeInject "InitPluginsDir"
+        iff_ (fileExists "$APPDATA/daml") $ do
+            _ <- messageBox [MB_OK] "DAML SDK is already installed. Use `daml install latest` to upgrade."
+            abort "Existing installation detected"
```

Comment:
I like the idea but I think having an easy way to nuke the existing installation and overwrite it is valuable. We don't have an `uninstall` command yet. Is it possible to turn this into a question "do you want to remove existing installation?" instead.

Reply:
I originally was planning to do that but got too frustrated by nsis. I’ll take another look and fix it in a separate PR if I can get it to work.

Reply:
:+1:

Having said that I realize I'm not even sure if the windows installer does even nuke an existing installation? It probably should, like get.daml.com does.

Reply:
It doesn’t atm so this definitely doesn’t make things worse. It only gives a useful error message instead of the error popping up somewhere in the details later.

Reply:
Managed to get it to work and included it in this PR. It now shows a yes/no popup. If you click yes it will remove the existing installation, if you click no it will abort.

Reply:
Awesome! Thank you 🙇‍♀